README.md

```diff
@@ -0,0 +1,48 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# The Book Corner
+
+A one page app to add and remove books from a list.
+
+![Maths Muggles](./Screenshot.png)
+
+## Built With
+
+- HTML/CSS/JavaScript
+- Create React App
+- React Router
+
+## Live Demo
+
+[Netlify Live Demo](https://aesthetic-panda-300b53.netlify.app)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- Click on the top right green "code" button.
+- On the dropdown menu, choose "download with zip" button.
+- After download, extract the zip file and you have the project running on your machine.
```

Comment:
- [ ] (Optional) It would be nice to include how to install the project, like `npm install`, how to start the project like `npm start` and how to run a test like `npm test` in this section 👍